wordpress-mobile/WordPress-Android
# Prepublishing Nudges : Page editor doesn't display tags in bottom sheet

Fixes #11891 

## Solution

We are able to check if a `post` is a `page`. Once we are able to do that in the `ViewModel` we can utilize it to toggle visibility of the `Tags` option. 

## Testing
1.  Create a new post. 
2. Click Publish Now. 
3. Ensure you see `Tags` in the bottom sheet. 
-----------
1.  Create a new page. 
2. Click Publish Now. 
3. Ensure you don't see `Tags` in the bottom sheet. 

Page - No Tags | Post - Tags 
--------|-------
   <kbd><img src="https://user-images.githubusercontent.com/1509205/82394571-06f30c00-9a0f-11ea-9f27-050484e28404.gif" width="320"></kbd>    |   <kbd><img src="https://user-images.githubusercontent.com/1509205/82394578-0bb7c000-9a0f-11ea-9bdd-dd7099592306.gif" width="320"></kbd>

## Reviewing 
Only 1 reviewer is needed but anyone can review. 

## Submitter Checklist
